### PR TITLE
OpenMC

### PR DESCRIPTION
**Pull request by scopatz**
This provides an OpenMC package for conda. It does not include neutron cross sections (order 2 - 8 GBs) and therefore cannot run a comprehensive test suite, so simpler tests have been implemented instead.

### CODE REVIEW

**Comment by scopatz** [comment]
Also, perhaps @paulromano, @nelsonag, etc can recommend some other maintainers for this package. It would be great to have someone more involved in core dev to be involved!

**Comment by scopatz** [comment]
Any chance this could be merged, please?  Thanks!

**Comment by scopatz** [comment]
Same as yesterday!

**Comment by scopatz** [comment]
Or maybe I should pin @conda-forge/core

**Comment by ocefpaf** [comment]
Sorry @scopatz I made a few comments yesterday but I was fooled by GitHub's new interlace and the comments never made it into the review.

**Comment by scopatz** [comment]
Ahh yeah! That has happened to me.  The UI is too easy to misuse.

File: recipes/openmc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - pkg-config
+    - gcc
+    - libgcc
+    - libgfortran
```
Comment (scopatz) [inline recipes/openmc/meta.yaml]: I'll update this right away!

**Comment by scopatz** [comment]
Woohoo! This is building again.

**Comment by scopatz** [comment]
Thanks again @ocefpaf!

**Comment by ocefpaf** [comment]
> Thanks again @ocefpaf!

@scopatz due to the high noise level here feel free to ping me on gitter when you need a review/merge.

**Comment by scopatz** [comment]
Will do!

**Comment by nelsonag** [comment]
@scopatz I'd love to help support something as beneficial for the scientific software development as conda; however I sadly dont have the time nor conda installed at the moment.  If either of those two change, however, I'd be glad to help out.

**Comment by scopatz** [comment]
Well, this specific issue is just making sure that the OpenMC package compiles and the new releases get built.  For conda-forge, you don't actually need conda installed since everything is run on CI servers.  You just need to be able to edit a [bash script](https://github.com/conda-forge/openmc-feedstock/blob/master/recipe/build.sh) and a yaml file.  This only happens as often as OpenMC is released.

**Comment by scopatz** [comment]
I actually think it is much less work than maintaining a PPA, which OpenMC does, and more applicable since the packages here can apply to Linux, Mac, and Windows.

**Comment by paulromano** [comment]
Yeah, the PPA is a pain in the ass. I'd be happy to shift support to this.

**Comment by scopatz** [comment]
Ok,  I'll open a PR to add you as a feedstock maintainer @paulromano.  Let me know if you are also interested @nelsonag.

**Pull request merged**